scrapy/czlscrape/settings.py

```diff
@@ -89,3 +89,7 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+LOG_LEVEL = 'WARN'
+LOG_FORMAT = '%(asctime)s srv="czl-scrape" [%(thread)d] %(levelname)s %(name)s: %(message)s'
```

Comment:
@mgax `%(name)s` is in fact the name of the logger, which is the same with the module name only by convention. We could just use custom stuff in there instead of `__name__`, and make margelatu happy.

Yeah, `funcName` doesn't look much fun. I personally would like to see the fully qualified name in there, something like `czlscrape.spiders.dialog.DialogSpider.parse`, or maybe with the module name separate from the rest.

This just made me think. If the log entry comes from someplace else, e.g. the middleware, there's no way we can tell which spider it was if we use the usual `logger = logging.getLogger(__name__)`.